questions/qa-bidi-unicode-controls.en.html

```diff
@@ -164,22 +176,13 @@ <h2>Changing the base direction</h2>
           <td>&lt;/bdo&gt;</td>
           <td>used for RLO or LRO</td>
         </tr>
-        <tr>
-          <td>PDI</td>
-          <td>POP DIRECTIONAL ISOLATE</td>
-          <td>U+2069</td>
-          <td>end tag</td>
-          <td>used for RLI, LRI or FSI</td>
-        </tr>
       </tbody>
     </table>
     <p>These characters are invisible, although in some editors it may be possible to show symbols that represent them. You could also use character escapes to represent them, such as <code>&amp;#x2067;</code>, but in bidirectional source text you may find that the characters in the escape don't stay together. (See <a href="https://www.w3.org/TR/2014/NOTE-i18n-html-tech-bidi-20140603/#bidisource"><cite>Problems with bidirectional source text in markup</cite></a> for more on this.)</p>
-    <div class="sidenoteGroup">
-    <p>When demarcating the boundaries of the change in base direction, you really want to avoid what's inside the boundaries interacting with what's outside – ie. you want to <em>isolate</em> it. For this reason, in an ideal world you would want to follow the recommendation of the Unicode Standard to  use RLI and LRI,  and avoid using RLE and LRE. Browsers running on Blink and Gecko engines (eg. Chrome, Firefox, etc.) support the use of RLI/LRI, but unfortunately, at the time of writing, WebKit browsers (eg. Safari) still don't support them properly, so you will need to resort to a workaround which we will describe below in order to avoid these spillover effects.</p>
-    <aside class="sideinfonote"><p class="info">This is one area where markup has an advantage over control codes, since it looks like isolation will be supported for markup before it is for the control codes.</p></aside>
-    </div>
-    <p>FSI is in the same boat as RLI/LRI as far as browser support is concerned. There is a set of tests for these control codes, with <a href="https://www.w3.org/International/tests/repo/results/rli-etc">results for major browsers</a>.</p>
-    <p>The following example shows how these control characters could be used in
+
+<p>When demarcating the boundaries of the change in base direction, you really want to avoid what's inside the boundaries interacting with what's outside – ie. you want to <em>isolate</em> it. For this reason, in an ideal world you would want to follow the recommendation of the Unicode Standard to  use RLI and LRI,  and avoid using RLE and LRE.</p>
```

Comment:
We suddenly talk about "boundaries of change" and I find this kind of a leap--I get it, but maybe not the audience.

Perhaps:

```suggestion
<p>When demarcating the boundaries of an inline run of text, you really want to avoid that text interacting with text that's outside – that is, you want to <em>isolate</em> the inline text from the rest of the paragraph. For this reason, in an ideal world you would want to follow the recommendation of the Unicode Standard to use RLI and LRI, and avoid using RLE and LRE.</p>
```

Reply:
Changed.